setup.py

```diff
@@ -70,15 +76,16 @@ def read_file(*path):
     zip_safe=False,  # otherwise ZConfig can't see component.xml
     install_requires=[
         'perfmetrics',
-        'ZODB3>=3.7.0',
+        'ZODB3>=3.7.0,<3.11.0',
```

Comment:
Can we not remove this version pin? We were using the 1.6 code with latest ZODB3 and it should work fine.